src/app.module.ts

```diff
@@ -2,7 +2,12 @@ import { Module, MiddlewareConsumer, RequestMethod } from '@nestjs/common';
 import i18n from 'i18n';
 
 import Database from '@app/config/database';
+
+/**
+ * Import Middleware
+ */
 import { HeaderMiddleware } from '@app/middleware/header.middleware';
+import APMAgent from '@app/middleware/apm-agent.middleware';
```

Comment:
it's not a class anymore.
it should be `apmAgent`

Reply:
Yah, it just naming right ?